DigitalLearningSolutions.Web/Views/LearningMenu/Section/_DiagnosticAssessmentCard.cshtml

```diff
@@ -14,7 +14,7 @@
     </div>
     <div class="nhsuk-grid-row">
       <div class="nhsuk-grid-column-full">
-        <a class="nhsuk-button nhsuk-u-margin-bottom-0 nhsuk-u-margin-top-0">
+        <a class="nhsuk-button nhsuk-u-margin-bottom-0 nhsuk-u-margin-top-0" asp-action="Diagnostic">
```

Comment:
I noticed before when I was testing that this button is ignored by the screen reader. Is that still a problem now? 🙂

Reply:
The screen reader reads out "launch assessment" when hovering over the button (in Firefox). Is that the expected response?

Reply:
What specifically you want to test is moving through the screen using the navigation buttons and tab. Using tab should select all the clickable elements such as that button, and read it out as a button/link. There is information about how to test with a screen reader on our testing page: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing 🙂

Reply:
I've tested the webpage with the screen reader (on Firefox) and it reads "link launch assessment" when using the navigation buttons and "launch assessment link" when using the tab key which I believe means it's working as intended.

Reply:
Great 🙂